addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -350,6 +334,10 @@
 <template id="registration_event_question" name="Registration Event Question">
     <label t-out="question.title"/>
     <span t-if="question.is_mandatory_answer">*</span>
+    <t t-if="question.is_identification_question">
```

Comment:
Looks like "is_identification_question" is only used in tests and here in the end.

Can't we instead use input_type_by_question_type.get(question.question_type, 'some_default_value') ?

And in tests adapt a bit so that we don't need this field? Feels a bit overkill.